airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java

```diff
@@ -123,6 +123,8 @@
     private static final String KEY_RECAPTCHA_SITE_KEY = "ReCaptchaSiteKey";
     private static final String KEY_RECAPTCHA_SECRET_KEY = "ReCaptchaSecretKey";
 
+    private static final String KEY_GOOGLE_CAST_ENABLED = "GoogleCast";
```

Comment:
Minor nit: I would rather call it `GoogleCastEnabled` or something to that effect.